via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -158,46 +169,40 @@ export default function VideoPlayerApp({
   };
 
   return (
-    <div className="w-full flex">
-      <div
-        className={classnames(
-          // This column will grow in width per the parent flex container and
-          // allow the contained media (video) to scale with available space.
-          // The column flex layout established here ensures this container fills
-          // the full height of the parent flex container
-          'grow flex flex-col p-3'
-        )}
-      >
-        <YouTubeVideoPlayer
-          videoId={videoId}
-          play={playing}
-          time={timestamp}
-          onPlayingChanged={setPlaying}
-          onTimeChanged={setTimestamp}
-        />
-      </div>
+    <div
+      data-testid="app-container"
```

Comment:
I acknowledge that this component's layout is getting complicated. To help a tiny bit for now, I've attempted to comment the heck out of styling and add `data-testid`s to elements.